activesupport/lib/active_support/hash_with_indifferent_access.rb

```diff
@@ -316,4 +316,5 @@ def set_defaults(target) # :doc:
   end
 end
 
+# :stopdoc:
 HashWithIndifferentAccess = ActiveSupport::HashWithIndifferentAccess
```

Comment:
I'm fine with this change but if we remove the constant from the public API I'd rater remove the constant all together. We have it only for backwards compatibility at this point. I'll ask more opinions. @matthewd @jeremy what do you think?